qcodes_contrib_drivers/drivers/Lakeshore/Model_625.py

```diff
@@ -0,0 +1,524 @@
+    
+        # Add reset function
+        self.add_function('reset', call_cmd='*RST')
+        if reset:
+            self.reset()
+
+        # Add power supply parameters
+        self.add_parameter(name='current_limit',
+                           unit="A",
+                           set_cmd=lambda x: self._set_curent_limit(x),
+                           get_cmd=self._get_current_limit,
+                           get_parser=float,
+                           vals=Numbers(0, 60.1),
+                           docstring="Maximum output current"
+                           )
+        
+        self.add_parameter(name='voltage_limit',
+                           unit="V",
+                           set_cmd=lambda x: self._set_voltage_limit(x),
+                           get_cmd=self._get_voltage_limit,
+                           get_parser=float,
+                           vals=Numbers(0, 5),
+                           docstring="Maximum compliance voltage"
+                           )
+        
+        self.add_parameter(name='current_rate_limit',
+                           unit="A/s",
+                           set_cmd=lambda x: self._set_current_rate_limit(x),
+                           get_cmd=self._get_current_rate_limit,
+                           get_parser=float,
+                           vals=Numbers( 0.0001, 99.999),
+                           docstring="Maximum current ramp rate"
+                           )
+        
+        self.add_parameter(name='voltage',
+                           unit = 'V',
+                           set_cmd="SETV {}",
+                           get_cmd='RDGV?',
+                           get_parser=float,
+                           vals=Numbers(-5, 5)
+                           )
+        
+        self.add_parameter(name='current',
+                           unit = 'A',
+                           set_cmd="SETI {}",
+                           get_cmd='RDGI?',
+                           get_parser=float,
+                           vals=Numbers(-60, 60)
+                           )
+        
+        self.add_parameter(name='current_ramp_rate',
+                           unit = 'A/s',
+                           set_cmd="RATE {}",
+                           get_cmd='RATE?',
+                           get_parser=float
+                           )
+        
+        self.add_parameter(name='ramp_segments',
+                           set_cmd="RSEG {}",
+                           get_cmd='RSEG?',
+                           get_parser=float,
+                           val_mapping={'disabled': 0,
+                                        'enabled': 1}
+                           )
+        
+        self.add_parameter(name='persistent_switch_heater',
+                           set_cmd=lambda x: self._set_persistent_switch_heater_status(x),
+                           get_cmd=self._get_persistent_switch_heater_status,
+                           get_parser=float,
```

Comment:
looks like this should perhaps be `get_parser=int` ? 

Note that qcodes as a create_on_off_val_mapping that you can consider using here https://qcodes.github.io/Qcodes/api/utils/helpers.html?highlight=val_mapping#qcodes.utils.helpers.create_on_off_val_mapping

Reply:
I changed to int. 
Thanks for the hint about the on_off_val_mapping, but since it works like it is, I would leave it.